falcon/response.py

```diff
@@ -911,6 +910,10 @@ def append_link(
                 # Un-inline in case more values are supported in the future.
                 value += '; crossorigin="use-credentials"'
 
+        if link_extension is not None:
+            value += '; '
+            value += '; '.join([ext[0] + '=' + ext[1] for ext in link_extension])
```

Comment:
I guess here we could be more rigid by requiring a two-item object, using something like `for p, v in link_extension`

Reply:
done